isubit/luggage_roles
# LUGG-459 Access to unpublished content for Content Editors

Added dependency for view_unpublished module and added permission to view unpublished content for content editor and view unpublished people to people editor.

Testing:
- Pull down my changes
- Download view_unpublished module
- Go to "Configuration" and verify that content publisher has a permission to view unpublished content
- Verify that people editor has a permission to view unpublished people content

Next....
- Create a new user
- Give new user content editor role
- Create unpublished content of any type (I tried page)
- Login as new user
- Click content tab
- Look at unpublished node
- On admin account, remove content editor role and replace with people editor role
- Create unpublished people node
- Go back to new user
- Verify that you can view unpublished people node (you'll have to manually enter the URL)
